### PR TITLE
Real pages

### PR DESCRIPTION
**Pull request by Sladuca**
- redesign most of the pages to the point where they are at least somewhat usable for user testing
- add link to not-yet-existant "Apps" page in the future so people could use dapps within the wallet
- make it look pretty and give it a baf-esque color scheme (idk if BAF has "official" colors or not, but that's something to deal with later)

### CODE REVIEW

File: apps/frontend/src/App.svelte [hunk 1]
```diff
@@ -9,12 +9,13 @@
 
   const routesLoggedIn = {
     '/': Account,
-    '/approve-redirect/:opts': ApproveRedirect,
+    // '/approve-redirect/:opts': ApproveRedirect,
```
Comment (Lev-Stambler) [inline apps/frontend/src/App.svelte:12]: I think we should change this back for now, lets discuss

File: apps/frontend/src/components/Auth.svelte [hunk 2]
```diff
@@ -21,11 +22,12 @@
...
+  <div class="social-is">
+    <Button classExtra="w-12">
+      <svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 245 240"><style>.st0{fill:#FFFFFF;}</style><path class="st0" d="M104.4 103.9c-5.7 0-10.2 5-10.2 11.1s4.6 11.1 10.2 11.1c5.7 0 10.2-5 10.2-11.1.1-6.1-4.5-11.1-10.2-11.1zM140.9 103.9c-5.7 0-10.2 5-10.2 11.1s4.6 11.1 10.2 11.1c5.7 0 10.2-5 10.2-11.1s-4.5-11.1-10.2-11.1z"/><path class="st0" d="M189.5 20h-134C44.2 20 35 29.2 35 40.6v135.2c0 11.4 9.2 20.6 20.5 20.6h113.4l-5.3-18.5 12.8 11.9 12.1 11.2 21.5 19V40.6c0-11.4-9.2-20.6-20.5-20.6zm-38.6 130.6s-3.6-4.3-6.6-8.1c13.1-3.7 18.1-11.9 18.1-11.9-4.1 2.7-8 4.6-11.5 5.9-5 2.1-9.8 3.5-14.5 4.3-9.6 1.8-18.4 1.3-25.9-.1-5.7-1.1-10.6-2.7-14.7-4.3-2.3-.9-4.8-2-7.3-3.4-.3-.2-.6-.3-.9-.5-.2-.1-.3-.2-.4-.3-1.8-1-2.8-1.7-2.8-1.7s4.8 8 17.5 11.8c-3 3.8-6.7 8.3-6.7 8.3-22.1-.7-30.5-15.2-30.5-15.2 0-32.2 14.4-58.3 14.4-58.3 14.4-10.8 28.1-10.5 28.1-10.5l1 1.2c-18 5.2-26.3 13.1-26.3 13.1s2.2-1.2 5.9-2.9c10.7-4.7 19.2-6 22.7-6.3.6-.1 1.1-.2 1.7-.2 6.1-.8 13-1 20.2-.2 9.5 1.1 19.7 3.9 30.1 9.6 0 0-7.9-7.5-24.9-12.7l1.4-1.6s13.7-.3 28.1 10.5c0 0 14.4 26.1 14.4 58.3 0 0-8.5 14.5-30.6 15.2z"/></svg>
```
Comment (Lev-Stambler) [inline apps/frontend/src/components/Auth.svelte:29]: Could we have a separate component for icons, lets try to keep the svg numbers seperate from the larger pages

File: apps/frontend/src/components/History.svelte [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        <tr>
+            <th>Date</th>
+            <th>Recipient</th>
+            <th>Data</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            {#each history as item}
+            <td class="text-center border">{item.confirmed.toLocaleString()}</td>
```
Comment (Lev-Stambler) [inline apps/frontend/src/components/History.svelte:23]: indent

File: apps/frontend/src/components/Navbar.svelte [hunk 4]
```diff
@@ -1,12 +1,18 @@
 <script lang="ts">
     import { link } from "svelte-spa-router"
     import Button from "./base/Button.svelte"
-    const openSendTxModal = () => {}
+    import SendModal from "./SendModal.svelte"
+
```
Comment (Lev-Stambler) [inline apps/frontend/src/components/Navbar.svelte:5]: empty lines

File: apps/frontend/src/pages/ApproveRedirect.svelte [hunk 5]
```diff
@@ -31,7 +62,9 @@
       )}
     </Card>
   {/if} -->
-  Result!!
+  <!-- Result!! -->
```
Comment (Lev-Stambler) [inline apps/frontend/src/pages/ApproveRedirect.svelte:65]: We could just delete this ig

File: libs/trust-wallet-assets/src/lib/index.ts [hunk 6]
```diff
@@ -2,126 +2,134 @@ import axios from 'axios';
...
+  const url = getChainInfoUrl(chain);
+  try {
+    const res = await axios.get(url);
+    const { data } = res;
 
-        // remove binance shill link
-        if (data.research !== undefined) {
-            delete data.research;
-        }
+    // remove binance shill link
```
Comment (Lev-Stambler) [inline libs/trust-wallet-assets/src/lib/index.ts:117]: luv it

**Review by Lev-Stambler — CHANGES_REQUESTED** [review]
Looks great! Just a few small things!

File: apps/frontend/src/components/Auth.svelte (hunk 2 above)
Comment (Sladuca) [reply apps/frontend/src/components/Auth.svelte:29]: Sure, Icon component makes sense.

**Review by Lev-Stambler — APPROVED** [review]
Lets gooo. Looks gtm

**Pull request merged**